src/components/skills-quiz/GoalDropdown.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Form } from '@edx/paragon';
+import PropTypes from 'prop-types';
+import {
+  DROPDOWN_OPTION_CHANGE_CAREERS, DROPDOWN_OPTION_CHANGE_RULE, DROPDOWN_OPTION_GET_PROMOTED, DROPDOWN_OPTION_OTHER,
+} from './constants';
+
+const GoalDropdown = ({ currentGoal, setCurrentGoal }) => {
```

Comment:
@muhammad-ammar if you are asking about these props, 
These props are passed from the src/components/skills-quiz/SkillsQuizStepper.jsx that is parent component, we are using currentGoal there.